custom_components/youtube/manifest.json

```diff
@@ -1,6 +1,7 @@
 {
   "domain": "youtube",
   "name": "Youtube Sensor",
+  "version": "0.0.0",
   "documentation": "https://github.com/custom-components/youtube",
   "dependencies": [],
   "codeowners": ["@ludeeus"],

```

Comment:
You should add yourself here

Reply:
will do, thanks

Reply:
should I add myself to `.github/CODEOWNERS` file also?